icu4c/source/test/intltest/intltest.cpp

```diff
@@ -2173,22 +2196,25 @@ UBool IntlTest::assertNotEquals(const char* message,
     return TRUE;
 }
 
-// http://junit.sourceforge.net/javadoc/org/junit/Assert.html#assertEquals(java.lang.String,%20double,%20double,%20double)
-UBool IntlTest::assertEqualsNear(const char *message, double expected, double actual, double precision) {
-    double diff = std::abs(expected - actual);
-    double diffPercent = expected != 0? diff / expected : diff; // If the expected is equals zero, we
-
-    if (diffPercent > precision) {
-        errln((UnicodeString) "FAIL: " + message + "; got " + actual + "; expected " + expected);
-        return FALSE;
-    }
-#ifdef VERBOSE_ASSERTIONS
-    else {
-        logln((UnicodeString) "Ok: " + message + "; got " + expected);
-    }
-#endif
-    return TRUE;
-}
+// // TODO: the following link is misleading, since JUnit does absolute numbers and
```

Comment:
This comment block is to be deleted, if we want to stay as close to JUnit as possible. Otherwise, this CL is of very low importance.